### DISCUSSION TITLE
Cannot get private repo's when loggedin with personal access token

### DISCUSSION THREAD

**Discussion by Tazi0** [octokit/octokit.js#2753] [Q&A]
So I don't know if it's a bug or just my fault. I am trying to get all my (public and private) repositories, and I'm currently using the GHP token to gather my data like so:
```ts
await octokit.repos.listForUser({
    username: "Tazi0",
  });
```
This only returns my public repositories. When I searched around, I saw that using the search function returns private repo's if you're logged in as the user. I am (checked via `await octokit.users.getAuthenticated()`). But the search also just returned public repositories.

So I checked the requests headers. And it does correctly add the `Authentication: Token ...` in the header. So i'm stuck :/

**Comment by HectorGitt** [comment]
Facing same issue currently, setting visibility to private shows 0 repositories
